js/scripts.js

```diff
@@ -33,30 +33,41 @@ function signUp() {
+    finalResume.workField1 = document.querySelector("#workField1")[workField1.selectedIndex].text;
+    finalResume.workName1 = document.querySelector("#workName1").value;
+    finalResume.workSkills1 = document.querySelector("#workSkills1")[workSkills1.selectedIndex].text;
+    finalResume.workAddress1 = document.querySelector("#workAddress1").value;
+    finalResume.workCountry1 = document.querySelector("#workCountry1")[workCountry1.selectedIndex].text;
+    finalResume.workState1 = document.querySelector("#workState1")[workState1.selectedIndex].text;
+    finalResume.workCity1 = document.querySelector("#workCity1").value;
+    finalResume.workZip1 = document.querySelector("#workZip1").value;
+    finalResume.workingHere1 = document.querySelector("#workingHere1").checked;
 }
 
 function volunteerHistory() {
-    finalResume.volunteerTitle = document.querySelector("#volunteerTitle")[volunteerTitle.selectedIndex].text;
-    finalResume.volunteerOrg = document.querySelector("#volunteerOrg").value;
-    finalResume.volunteerSkills = document.querySelector("#volunteerSkills")[volunteerSkills.selectedIndex].text;
-    finalResume.volunteerCity = document.querySelector("#volunteerCity").value;
-    finalResume.volunteerYears = document.querySelector("#volunteerYears")[volunteerYears.selectedIndex].text;
+    finalResume.volunteerTitle1 = document.querySelector("#volunteerTitle1")[volunteerTitle1.selectedIndex].text;
+    finalResume.volunteerOrg1 = document.querySelector("#volunteerOrg1").value;
+    finalResume.volunteerSkills1 = document.querySelector("#volunteerSkills1")[volunteerSkills1.selectedIndex].text;
+    finalResume.volunteerCity1 = document.querySelector("#volunteerCity1").value;
+    finalResume.volunteerYears1 = document.querySelector("#volunteerYears1")[volunteerYears1.selectedIndex].text;
 }
 
 function educationHistory() {
-    finalResume.educationLevel = document.querySelector("#educationLevel")[educationLevel.selectedIndex].text;
-    finalResume.educationSchool = document.querySelector("#educationSchool").value;
-    finalResume.educationStatus = document.querySelector("#educationStatus")[educationStatus.selectedIndex].text;
-    finalResume.educationField = document.querySelector("#educationField").value;
+    finalResume.educationLevel1 = document.querySelector("#educationLevel1")[educationLevel1.selectedIndex].text;
+    finalResume.educationSchool1 = document.querySelector("#educationSchool1").value;
+    finalResume.educationStatus1 = document.querySelector("#educationStatus1")[educationStatus1.selectedIndex].text;
+    finalResume.educationField1 = document.querySelector("#educationField1").value;
+}
+
+//Function to Update Ids of all inputs when forms are cloned
+function updateIds(clone, newNum) {
+    let inputs = clone.querySelectorAll("*");
+    console.log(inputs);
+    inputs.forEach(function(input) {
+        if (input.id) {
+            input.setAttribute("id", input.id.slice(0, -1) + newNum);
```

Comment:
Yooooo!  This is actually pretty amazing.  Excellent usage of slice here.  I hadn't even thought of that!

Reply:
Awesome, thanks! Was slightly frustrating at first and I'm super proud this fix worked haha.